torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -349,15 +349,15 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                 if args[0] >= 2:
                     # step = 1
                     new_args = (args[0],)
-                    new_kwargs = {"end": args[1], "step": 1, "device": TtnnDevice()}
+                    new_kwargs = {"stop": args[1], "step": 1, "device": TtnnDevice()}
                     new_node = g.call_function(ttnn.arange, args=new_args, kwargs=new_kwargs)
                     new_nodes.append(new_node)
             if node.target == torch.ops.aten.arange.start_step:
                 # NOTE(kevinwuTT): ttnn.arange does not support starting values smaller than 2 currently
                 if args[0] >= 2:
                     new_args = (args[0],)
                     new_kwargs = {
-                        "end": args[1],
+                        "stop": args[1],
```

Comment:
I see in the code that right now stop is not a kw argument, but a positional one. 

https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/pybind11/operations/creation.hpp#L167-L197

We will change, but please adjust for now.

Reply:
@jdh8 CI fails this test, which tells me you did not run the test locally yet. I ask to please run tests locally too

Reply:
I ran the tests locally and they passed.  I suspect that the difference was caused by difference in `tt-metal` versions, and I'm running into issues while updating `tt-metal`.
```
FAILED: tools/watcher_dump 
: && /usr/bin/clang++-17  -O3   tt_metal/tools/watcher_dump/CMakeFiles/watcher_dump.dir/watcher_dump.cpp.o  -o tools/watcher_dump  -Wl,-rpath,/home/ubuntu/jdh8/tt-metal/build_Release/tt_metal:/home/ubuntu/jdh8/tt-metal/build_Release/lib:/home/ubuntu/jdh8/tt-metal/build_Release/tt_metal/third_party/umd/device/simulation:/home/ubuntu/jdh8/tt-metal/build_Release/_deps/nanomsg-build:/home/ubuntu/jdh8/tt-metal/build_Release/_deps/libuv-build  tt_metal/libtt_metal.so  lib/libdevice.so  tt_metal/third_party/umd/device/simulation/libsimulation.so  _deps/nanomsg-build/libnng.so.1.8.0  _deps/libuv-build/libuv.so.1.0.0  -lrt  -ldl  -lz  -latomic  -lhwloc  -lnuma  -lc++  /usr/lib/x86_64-linux-gnu/libc++abi.so  -lpthread  lib/libgtest_main.so.1.13.0  lib/libgtest.so.1.13.0  -lpthread && :
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `tt_SimulationDevice::tt_SimulationDevice(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)'
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `tt_ClusterDescriptor::create_for_grayskull_cluster(std::__1::set<int, std::__1::less<int>, std::__1::allocator<int> > const&, std::__1::vector<int, std::__1::allocator<int> > const&)'
/usr/bin/ld: lib/libdevice.so: undefined reference to `YAML::detail::node_data::convert_to_map(std::shared_ptr<YAML::detail::memory_holder> const&)'
/usr/bin/ld: lib/libdevice.so: undefined reference to `YAML::detail::node_data::set_scalar(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `YAML::LoadFile(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)'
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `format_node(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >)'
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `tt_SiliconDevice::tt_SiliconDevice(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::set<int, std::__1::less<int>, std::__1::allocator<int> > const&, unsigned int const&, std::__1::unordered_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, int, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > >, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const, int> > > const&, bool, bool, bool, std::__1::unordered_map<int, unsigned int, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, unsigned int> > >)'
/usr/bin/ld: lib/libdevice.so: undefined reference to `YAML::LoadFile(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/usr/bin/ld: tt_metal/libtt_metal.so: undefined reference to `tt_ClusterDescriptor::create_from_yaml(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&)'
clang++-17: error: linker command failed with exit code 1 (use -v to see invocation)
[58/785] Building CXX object ttnn/CMakeFiles/ttnn.dir/cpp/ttnn/operations/normalization/softmax/softmax_pybind.cpp.o
ninja: build stopped: subcommand failed.
```

Reply:
Repo specifies exact metal dependency in requirements-dev.txt and I think there were no updates since M0 PR was merged